js/client-chat-tournament.js

```diff
@@ -266,7 +266,7 @@
 					var formatName = window.BattleFormats && BattleFormats[data[0]] ? BattleFormats[data[0]].name : data[0];
 					var type = data[1];
 					this.room.$chat.append("<div class=\"notice tournament-message-create\">" + Tools.escapeHTML(formatName) + " " + Tools.escapeHTML(type) + " Tournament created.</div>");
-					this.room.notifyOnce("Tournament created", "Room: " + this.room.title + "\nFormat: " + formatName + "\nType: " + type, 'tournament-create');
+					if (!onInit) this.room.notifyOnce("Tournament created", "Room: " + this.room.title + "\nFormat: " + formatName + "\nType: " + type, 'tournament-create');
```

Comment:
In theory, we can, it'd just be more code.

I think the simplest approach is, if the tournament has already started, remove the notification.

Except clearing the `notifyOnce` notification would also clear highlights, so maybe we should also track a `notifyOncePriority` which is `1` for tournaments and `2` for highlights, and only clear it if it's `1`.